packages/hardware-trezor/src/transformers/auxiliaryData.ts

```diff
@@ -1,6 +1,10 @@
 import * as Crypto from '@cardano-sdk/crypto';
 import * as Trezor from '@trezor/connect';
+import { Transform } from '@cardano-sdk/util';
 
-export const mapAuxiliaryData = (auxiliaryDataHash: Crypto.Hash32ByteBase16): Trezor.CardanoAuxiliaryData => ({
-  hash: auxiliaryDataHash
+export const mapAuxiliaryData: Transform<Crypto.Hash32ByteBase16, Trezor.CardanoAuxiliaryData> = (
+  auxiliaryDataHash
+) => ({
+  cVoteRegistrationParameters: undefined,
+  hash: auxiliaryDataHash // Voting is not handled for now
```

Comment:
This comment applies to line 8 right?

Reply:
@rhyslbw fixed in [e69f4b6](https://github.com/input-output-hk/cardano-js-sdk/pull/1096/commits/e69f4b619fd1ee41c5d5189beea8442a30c375df)